src/Cms/PageActions.php

```diff
@@ -506,10 +506,14 @@ public function createNum(int $num = null): int
 
                 return $num;
             default:
+                // get instance with default lanuage
+                $app = $this->kirby()->clone();
+                $app->setCurrentLanguage();
```

Comment:
@lukasbestle just wondering: this does set the default language as current, but should we make it even more explicit (actually passing the default language)?

Reply:
I thought the same when reviewing, but in the end it should be fine. Maybe we could explicitly pass `null` to make it clear that the default language is intended.